client/package.json

```diff
@@ -91,6 +92,7 @@
     "react-dropzone": "^3.13.2",
     "react-intl": "^2.3.0",
     "react-overlays": "^0.7.0",
+    "react-player": "^0.24.2",
```

Comment:
We should make these libraries async if they're only used in the video page. That is keep these two libs separete from the main bundle and only load then on video page. You can check webpack documentation for async import, and there are some examples in the existing code too.

Reply:
Alright, that sounds like a better way to do it, yes. Will look into ti!